deploy/role.yaml

```diff
@@ -40,6 +40,18 @@ rules:
   - statefulsets/finalizers
   verbs:
   - '*'
+- apiGroups:
+  - "route.openshift.io"
+  resources:
+  - routes
```

Comment:
@rigazilla looks like we also need following resource
- routes/custom-host

Reply:
right, user can also set the host. Thanks @dmvolod !